[2025-09-10T06:37:30.711+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-10T06:37:30.786+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_model_dag_batch_iterative_85000.wait_for_predict_api manual__2025-09-10T06:37:23.443644+00:00 [queued]>
[2025-09-10T06:37:30.830+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_model_dag_batch_iterative_85000.wait_for_predict_api manual__2025-09-10T06:37:23.443644+00:00 [queued]>
[2025-09-10T06:37:30.832+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-10T06:37:30.888+0000] {taskinstance.py:2330} INFO - Executing <Task(HttpSensor): wait_for_predict_api> on 2025-09-10 06:37:23.443644+00:00
[2025-09-10T06:37:30.953+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'train_model_dag_batch_iterative_85000', 'wait_for_predict_api', 'manual__2025-09-10T06:37:23.443644+00:00', '--job-id', '419', '--raw', '--subdir', 'DAGS_FOLDER/train_dag.py', '--cfg-path', '/tmp/tmp1etbreeg']
[2025-09-10T06:37:30.939+0000] {standard_task_runner.py:63} INFO - Started process 403 to run task
[2025-09-10T06:37:30.974+0000] {standard_task_runner.py:91} INFO - Job 419: Subtask wait_for_predict_api
[2025-09-10T06:37:29.450+0000] {task_command.py:426} INFO - Running <TaskInstance: train_model_dag_batch_iterative_85000.wait_for_predict_api manual__2025-09-10T06:37:23.443644+00:00 [running]> on host 783f489a94dd
[2025-09-10T06:37:29.853+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_model_dag_batch_iterative_85000' AIRFLOW_CTX_TASK_ID='wait_for_predict_api' AIRFLOW_CTX_EXECUTION_DATE='2025-09-10T06:37:23.443644+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-10T06:37:23.443644+00:00'
[2025-09-10T06:37:29.857+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-10T06:37:29.937+0000] {http.py:130} INFO - Poking: /health
[2025-09-10T06:37:29.946+0000] {base.py:84} INFO - Using connection ID 'predict_api' for task execution.
[2025-09-10T06:37:30.069+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2025-09-10T06:37:30.070+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-10T06:37:30.106+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=train_model_dag_batch_iterative_85000, task_id=wait_for_predict_api, run_id=manual__2025-09-10T06:37:23.443644+00:00, execution_date=20250910T063723, start_date=20250910T063730, end_date=20250910T063730
[2025-09-10T06:37:30.197+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-10T06:37:30.338+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-10T06:37:30.344+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
